app/src/main/java/com/cops/iitbhu/previewer/VideoToBitmapActivity.kt

```diff
@@ -0,0 +1,40 @@
+package com.cops.iitbhu.previewer
+
+import android.net.Uri
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.lifecycle.lifecycleScope
+import com.cops.iitbhu.previewer.databinding.ActivityVideoToBitmapBinding
+import com.cops.iitbhu.previewer.lib.Previewer
+
+class VideoToBitmapActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityVideoToBitmapBinding
+
+    private val activityResultLauncherForVideoUri =
+        registerForActivityResult(ActivityResultContracts.GetContent()) { videoUri: Uri ->
+            lifecycleScope.launchWhenCreated {
```

Comment:
No need to create a coroutine here. `getThumbnailFromLocalVideoUri` is a non-suspending function and call be called directly.

Reply:
done